src/apiGatewayUtilities/index.ts

```diff
@@ -65,4 +65,36 @@ export function add500Handler(func: (event: apigateway.Request) => Promise<apiga
       return buildApiResponse(500, error);
     }
   };
+}
+
+export async function query(
```

Comment:
Let's move these to another module, database operations have little to do with API gateway

Reply:
ok